exe/solr_wrapper

```diff
@@ -100,14 +110,15 @@ instance = SolrWrapper.instance(options)
 
 case command
 when 'clean'
-  if instance.started?
-    $stderr.puts "Please stop solr before cleaning"
-    exit 1
-  end
+  return if ENV['RAILS_ENV'] && ENV['RAILS_ENV'] == 'production'
   $stderr.puts "cleaning #{instance.instance_dir}..."
-  instance.remove_instance_dir!
+  instance.clean!
```

Comment:
The `clean!` calls `stop` so that's the big diff.  Given that, then it's not necessary to check if it's stopped, it will get stopped.